visualSHARK/views.py

```diff
@@ -831,6 +849,8 @@ def test_connection_worker(self, request, pk=None):
 class IssueLabelSet(APIView):
 
     def get(self, request):
+        if (not request.user.has_perm('visualSHARK.view_issue_labels')):
```

Comment:
This should be implemented like here: https://www.django-rest-framework.org/api-guide/viewsets/#introspecting-viewset-actions
Otherwise we might miss stuff like delete(), partial_update() which is exposed view the ReST API.
The parentheses can also be removed.

Reply:
That is not working since we are using custom permissions. If you like to use this pattern, we should create a new Permission class and override the  def has_permission(self, request, view) method. Thus the mapping of permissions to views is done in the has_permission method, which would be very bad..  

See details here: https://stackoverflow.com/questions/23716855/django-rest-framework-custom-permissions-per-view